namui/src/namui/skia/web/paint.rs

```diff
@@ -1,5 +1,5 @@
 use super::*;
-use crate::namui;
+
```

Comment:
```suggestion
```
I think it would be better to delete the blank line between imports. For the lint to work

Reply:
Fixed in d48d9fd574bd10dfc30f5a1117e2490107277c4dc